### PR TITLE
use / for purge url and fix document id

### PR DESCRIPTION
**Pull request by tom-james-watson**

### CODE REVIEW

File: src/tasks/highwinds.js [hunk 1]
```diff
@@ -175,10 +175,7 @@ export default class PurgeDocuments {
         Promise
           .all([
             highwinds.purgeByManyTags(digitalAssetSids),
-            highwinds.purgeByUrl('', {
-              invalidateOnly: true,
-              recursive: true
-            })
+            highwinds.purgeByUrl('/', {recursive: true})
```
Comment (tom-james-watson) [inline src/tasks/highwinds.js]: It defaults to `true` https://github.com/f12/highwinds-node/blob/master/lib/highwinds.js#L301

**Comment by tom-james-watson** [comment]
[Confirm that we are using invalidateOnly on all purge calls in the API](https://trello.com/c/TuaXFNPa/388-confirm-that-we-are-using-invalidateonly-on-all-purge-calls-in-the-api)

File: src/tasks/highwinds.js (hunk 1 above)
Comment (chrisabrams) [inline src/tasks/highwinds.js]: Remove the forward slash, for some reason that doesn't work.

Comment (tom-james-watson) [inline src/tasks/highwinds.js]: Done!

**Pull request merged**